### DISCUSSION TITLE
区块浏览器转账记录

### DISCUSSION THREAD

**Discussion by zoukz115** [mangodager/smartx-dotnet#18] [General]
用官网区块浏览器查询转账记录，每一条的转账记录的交易类型，转入转出标注反了。明明是转出，交易类型却是转入。
![image](https://user-images.githubusercontent.com/71767976/173257936-b092550c-3dfc-4f80-aef6-6653ac0a5a45.png)

**Comment by mangodager** [comment]
中文已经不被支持

**Reply by zoukz115** [reply]
英文的意思也是反的啊
看不懂问题在哪里吗？

**Reply by zoukz115** [reply]
转入记录，英文标注是to，转出记录，标注为from。这难道是正确的吗？
